library/ZendService/Twitter/Response.php

```diff
@@ -152,4 +160,52 @@ public function toValue()
     {
         return $this->jsonBody;
     }
+
+    /**
+     * Retun the array of response rate info
+     *
+     * @return array
+     */
+    public function getRateLimit()
+    {
+        return $this->rateLimit;
+    }
+
+    /**
+     * Populates the object with info. This can possibly called from the
```

Comment:
Please don't make this method public: it is trivial to just call `__construct($httpResponse)`, so please don't allow the user to create an empty `Response` object, which may result in bugs if he forgets to call this method afterwards